daemon/container_operations.go

```diff
@@ -851,39 +900,6 @@ func (daemon *Daemon) normalizeNetMode(ctr *container.Container) error {
 	return nil
 }
 
-func (daemon *Daemon) initializeNetworking(ctx context.Context, cfg *config.Config, ctr *container.Container) error {
```

Comment:
Looks like this patch swapped the order of `initializeNetworking` and  `allocateNetwork`, which makes the diff harder to read; would you mind changing the order back to reduce the diff?

Reply:
The functions weren't exactly swapped, code moved between them. I'd tried to minimise the diff in terms of lines-changed.

Maybe it's easier to understand a bigger diff, but without moving function declarations - just moving more lines between functions?

So I've made the change (can put it back if needed). The diff has gone from:
`1 file changed, 58 insertions(+), 42 deletions(-)` to
`1 file changed, 68 insertions(+), 52 deletions(-)`.